src/main/java/fr/raluy/chocoratage/KeyLogger.java

```diff
@@ -27,38 +27,44 @@ public KeyLogger() {
         }
     }
 
+    /**
+     * keyCode is defined here, not keyChar
```

Comment:
I dont understand what you meant here.

Reply:
depending on the context EITHER keyCode (nativeKeyPressed, nativeKeyReleased) OR keyChar (nativeKeyTyped) is defined in the nativeKeyEvent object. It's not even the same object and it's confusing, so I commented on that to remember next time I come back in this area.